ably/proto_http.go

```diff
@@ -23,11 +24,20 @@ var goRuntimeIdentifier = func() string {
 	return fmt.Sprintf("%s/%s", libraryName, runtime.Version()[2:])
 }()
 
-var ablyAgentIdentifier = func() string {
+func ablyAgentIdentifier(agents map[string]string) string {
+	elems := []string{
```

Comment:
Does elems stand for something or just a variable name? Maybe a bit more descriptive

Reply:
Sure renamed `identifiers`